pkg/vsphere/session/session.go

```diff
@@ -187,6 +188,10 @@ func (s *Session) Create(ctx context.Context) (*Session, error) {
 
 // Connect establishes the connection for the session but nothing more
 func (s *Session) Connect(ctx context.Context) (*Session, error) {
+	op := trace.FromContext(ctx, "Connect")
+
+	op.Debugf("Creating VMOMI session with thumbprint %s", s.Config.Thumbprint)
```

Comment:
`Thumbprint` is referenced as `s.Thumbprint` on line 223 - can we use consistent reference, otherwise it looks like there's two thumbprints instead of an embedded struct.